mineinabyss-features/src/main/kotlin/com/mineinabyss/displayLocker/DisplayLockerFeature.kt

```diff
@@ -0,0 +1,84 @@
+package com.mineinabyss.displayLocker
+
+import com.mineinabyss.components.armorstandlock.LockArmorStand
+import com.mineinabyss.components.playerData
+import com.mineinabyss.geary.papermc.access.toGeary
+import com.mineinabyss.geary.papermc.store.encodeComponentsTo
+import com.mineinabyss.idofront.commands.arguments.stringArg
+import com.mineinabyss.idofront.commands.extensions.actions.playerAction
+import com.mineinabyss.idofront.entities.toPlayer
+import com.mineinabyss.idofront.messaging.broadcast
+import com.mineinabyss.idofront.messaging.error
+import com.mineinabyss.idofront.messaging.info
+import com.mineinabyss.idofront.messaging.success
+import com.mineinabyss.idofront.plugin.registerEvents
+import com.mineinabyss.mineinabyss.core.AbyssFeature
+import com.mineinabyss.mineinabyss.core.MineInAbyssPlugin
+import com.mineinabyss.mineinabyss.core.commands
+import kotlinx.serialization.SerialName
+import net.minecraft.util.DirectoryLock.LockException
+import org.bukkit.Bukkit
+import org.bukkit.entity.Player
+
+
+@kotlinx.serialization.Serializable
+@SerialName("lock_display_item")
+class DisplayLockerFeature: AbyssFeature {
+    override fun MineInAbyssPlugin.enableFeature() {
+        registerEvents(DisplayLockerListener())
+
+        commands {
+            mineinabyss {
+                "lock"(desc="display locker commands") {
+                    val player = sender as Player
+                    val entity = player.playerData.recentRightclickedEntity
+                    val locked = entity?.toGeary()?.get<LockArmorStand>()
```

Comment:
Can you rename the component to fit aswell

Reply:
fixed in [ab91c86](https://github.com/MineInAbyss/MineInAbyss/pull/159/commits/ab91c86505a9eb3a171879287a53a93c13223c87)